tests/fixture/e2e/helpers.go

```diff
@@ -49,6 +52,13 @@ const (
 	PrivateNetworksDirName = "private_networks"
 )
 
+// Create a new private key.
+func NewPrivateKey(tc tests.TestContext) *secp256k1.PrivateKey {
+	key, err := secp256k1.NewPrivateKey()
+	require.NoError(tc, err)
+	return key
+}
```

Comment:
I could pull this out into a separate PR to avoid mixing functional + style changes... But I ended up cleaning up the tests so that I could better understand what they were doing (so that I could fix them for dynamic fees) and ended up making this change as part of it... Will defer to reviewer opinion here.